<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>641</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/login/image/a07.png);
border-top-left-radius:20px;
border-bottom-left-radius:20px;
border-top-right-radius:20px;
border-bottom-right-radius:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton{
	border:none
}
#pushButton:focus{
	
	color: rgb(216, 216, 216);
}</string>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_2{
	border:none
}
#pushButton_2:focus{
	
	color: rgb(216, 216, 216);
}</string>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>391</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(236, 236, 236, 50);</string>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>331</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgb(0,0,0);
border-radius:8px;
</string>
    </property>
    <property name="placeholderText">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>331</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgb(0,0,0);
border-radius:8px;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>密码：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>331</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_3{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	border:1px solid rgb(0, 0, 0);
	border-radius:4px;
}

#pushButton_3:hover{
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

#pushButton_3:pressed{
	padding-top:5px;
	padding-left:5px;
}</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Resource/resource.qrc">
      <normaloff>:/login/ico/Login-PasswordIsNotVisible.png</normaloff>
      <normalon>:/login/ico/Login-PasswordIsVisible.png</normalon>
      <disabledoff>:/login/ico/Login-PasswordIsNotVisible.png</disabledoff>
      <disabledon>:/login/ico/Login-PasswordIsVisible.png</disabledon>:/login/ico/Login-PasswordIsNotVisible.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>40</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Resource/resource.qrc">
     <normaloff>:/login/ico/close.png</normaloff>:/login/ico/close.png</iconset>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>391</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(236, 236, 236, 50);</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>331</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgb(0,0,0);
border-radius:8px;
</string>
    </property>
    <property name="placeholderText">
     <string>输入用户名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>331</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgb(0,0,0);
border-radius:8px;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>输入密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>331</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgb(0,0,0);
border-radius:8px;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>确认密码：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>331</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_5{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	border:1px solid rgb(0, 0, 0);
	border-radius:4px;
}

#pushButton_5:hover{
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

#pushButton_5:pressed{
	padding-top:5px;
	padding-left:5px;
}</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
  </widget>
  <zorder>label</zorder>
  <zorder>widget</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>widget_3</zorder>
  <zorder>widget_2</zorder>
 </widget>
 <resources>
  <include location="../Resource/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
